### ISSUE TITLE
Run-ahead per-game configuration issues.

### ISSUE THREAD

**Issue by n1ghttrap** [libretro/RetroArch#7024]
I have RA set up to not run ahead by default.  I have various cores set to run ahead 1 frame.  When I try to save a per-game configuration with a run-ahead off (because it might be more demanding or not work right), it won't write a config file with run-ahead marked off.  It will save my frames of run ahead count if I make that change, but will not make a flag for run-ahead off.  It's looking to my general RA settings for the difference, and ignoring the core's override when it writes the game override, I believe.

Was wondering if this issue could be resolved so I could have the core run-ahead by default, but per game disabled if I specify that.  Thanks for your time.

**Comment by hizzlekizzle** [comment]
Yeah, the only other option is to manually add the line to your per-game overrides, though they'll get wiped out if you save another per-game override over the top of it.

**Comment by n1ghttrap** [comment]
I engineered a solution of sorts, in case anyone's curious:  I saved run-head as true with 0 frames to RA's default settings.  Then made a core save with run-ahead +1, and the game with run-ahead false.  This made it work as it should.

**Comment by fr500** [comment]
This is why I wanted to remove "nesting"
But @Tatsuya79 said he used nested overrides so I left it as is.
